openmp/libomptarget/plugins-nextgen/common/src/Utils/ELF.cpp

```diff
@@ -283,3 +295,44 @@ Expected<const void *> utils::elf::getSymbolAddress(
 
   return ELFFile.base() + Offset;
 }
+
+template <class ELFT>
+static Expected<std::optional<StringRef>>
+findSymbolInImageImpl(const object::ELFObjectFile<ELFT> &ELFObj,
+                      StringRef Name) {
+  // Search for the symbol by name.
```

Comment:
Nit, a lot of these comments are just restating what is generally observable from the code. I.e. `getSymbol(ELFObj, Name)` implies we're looking up a symbol by name.

Reply:
Left in a single comment describing the return value, removed all the others.

Reply:
Thanks, looks good.